src/NotepadNext/dialogs/MainWindow.cpp

```diff
@@ -663,7 +663,7 @@ void MainWindow::openFileList(const QStringList &fileNames)
             QFileInfo fileInfo(filePath);
 
             if (!fileInfo.isFile()) {
-                auto reply = QMessageBox::question(this, "Create File", QString("<b>%1</b> does not exist. Do you want to create it?").arg(filePath));
+                auto reply = QMessageBox::question(this, tr("Create File"), QString(tr("<b>%1</b> does not exist. Do you want to create it?")).arg(filePath));
```

Comment:
QString() can be removed here:

```suggestion
                auto reply = QMessageBox::question(this, tr("Create File"), tr("<b>%1</b> does not exist. Do you want to create it?").arg(filePath));
```

Reply:
done